### PR TITLE
Upgraded Jekyll version utilized in Dockerfile

### PR DESCRIPTION
**Pull request by dpeachpeach**
Successfully served data-and-lab locally. I looked for opportunities to upgrade Jekyll where I could and only saw an improvement I could make in the Dockerfile.
Something I have a question about is the way data-and-lab runs without utilizing a Gemfile (All the docs I found online stated that a gemfile was essential and in the practice jekyll website I made I utilized a gemfile). 
Kepler.Gl integration next.

### CODE REVIEW

**Comment by dpeachpeach** [comment]
I didn't merge using an issue, that was my bad, going to do so for Kepler.gl integration.

File: Dockerfile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 # Pull down a Jekyll base image (which uses ruby:2.7.1-alpine3.11)
 
 # dev-stage
-FROM jekyll/jekyll:4.0
+FROM jekyll/jekyll:4.2.2
```
Comment (lixun910) [inline Dockerfile:4]: @dpeachpeach Did you try it on Apple M1? It seems not working correctly on arm64. Thanks!

**Comment by lixun910** [comment]
@dpeachpeach I can't get local dev working with docker. Instead, I just upgraded gulp to v4, so with node v12, one can run local dev using `gulp`. You need to rerun `npm install` and then `gulp`. Another simple way to do: you can just run `jekyll build` and `jekyll serve` to get local dev working. 

Let me know what you think. Thanks!

File: package.json [hunk 2]
```diff
@@ -33,7 +33,10 @@
     "rupture": "^0.6.1"
   },
   "repository": {
-    "type" : "git",
-    "url" : "https://github.com/GeoDaCenter/data-and-lab.git"
+    "type": "git",
+    "url": "https://github.com/GeoDaCenter/data-and-lab.git"
+  },
+  "volta": {
+    "node": "12.22.12"
```
Comment (lixun910) [inline package.json:40]: You can use volta or nvm to pin a specific node version

**Review by lixun910 — APPROVED** [review]
You can merge it once you verify it on your side. Thanks, @dpeachpeach !

**Comment by dpeachpeach** [comment]
Thanks Xun! I'm going to verify it right now

**Comment by lixun910** [comment]
@dpeachpeach If everything works as expected, you can click the green "Squash and merge" button. Thanks!

**Pull request merged**